databaseconnect.py

```diff
@@ -187,7 +189,7 @@ def clear_table(table_name):
 
         if input("Enter 'Y' to confirm cleaning of BOTH tables: ") in ("Y", "y"):
             for table in tables_to_be_cleaned:
-                cur.execute(f"DELETE FROM {table}")
+                cur.execute("DELETE FROM {table}")
```

Comment:
Why did you delete this `f`?
This would not work without the 'f'. Please refrain from making unnecessary changes not related to your original issue.